src/atoms/Button/buttons.module.scss

```diff
@@ -30,36 +30,25 @@ $brand-white: color('primary-2');
   width: 100%;
   outline: none;
   white-space: normal;
-  border-radius: 0;
 
   &:active,
   &:focus {
     outline: none;
   }
 
-  .icon {
-    path {
-      transition: fill 300ms ease-out;
-      fill: $brand-white;
-    }
+  .icon path {
```

Comment:
It seems like someone had tried to style the button icons at one point, but this approach isn't exactly enough anymore. Since the `<Icon>` in a `<Button>` now renders the svg to the DOM, this styling works!